### ISSUE TITLE
Add flag that will force rollback

### ISSUE THREAD

**Issue by alexander-gorelik** [chaostoolkit/chaostoolkit#176]
### The problem

I have an experiment that runs for half of an hour, After 5 mins I get an email from the Production team that something bad happened in prod(not as a result of my experiment) that causes a really big impact on users and they are trying to solve it.

The demand from me is to stop the execution of my experiment for not adding additional chaos to prod that already suffering.

Currently, CTK has only one option to stop the experiment by throwing `InterruptExecution`

In this case, rollback won't be executed.

### Feature needed

Add `force` flag to actions/probes in rollback.
`force` will be defaulted to false to maintain CTK backward compatibility.
 
```
{ 
    "steady-state-hypothesis": {
    },
    "method": [
    ],
    "rollbacks": [
        {
            "type": "action",
            "name": "swap-to-vald-cert",
            "force": "true"
        },
        {
            "type": "action",
            "name": "swap-to-vald-cert",
        }
    ]
}


```

Thanks!

**Comment by Lawouach** [comment]
I'm looking into this and I'm considering two options:

* add a flag to the run command
* add a flag to the settings file

The former is more explicit but I am frugal when it comes to adding new command line flags if I can help it. This makes helps hard to go through and makes you wonder if you should try a combination of stuff.

The latter requires one more call before and is enabled across all `chaos run` calls of the user.

**Comment by Lawouach** [comment]
I will likely go for the flag for now. The latter requires a deeper discussion about the settings supporting command line flags across the board.

**Comment by Lawouach** [comment]
Related to #177

**Comment by Lawouach** [comment]
@alexander-gorelik could you please try this branches for me:

```
$ pip install -U git+https://github.com/chaostoolkit/chaostoolkit-lib.git#chaostoolkit-lib
$ pip install -U git+https://github.com/chaostoolkit/chaostoolkit.git@issue176#chaostoolkit
```

This will add a new flag to `chaos run` as per https://github.com/chaostoolkit/chaostoolkit/blob/issue176/CHANGELOG.md#added

Let me know your thoughts.

**Comment by alexander-gorelik** [comment]
Looks good and works :-)

You have a small typo.

"Rollbacks **werre** explicitly requested to be played only if the experiment deviated. Since this is not the case, we will not play them."

I think this feature will be used by experiments that need high-level control of the rollback. 

What is the use case to use `--rollback-strategy=never` you thought about?

**Comment by Lawouach** [comment]
It could be interesting at times to see how the system resolves itself. Or maybe you want to explore further the system before it is brought back to normal.

**Comment by alexander-gorelik** [comment]
It's kinda the same as not adding the rollback section at all :-)

**Comment by Lawouach** [comment]
Totally, it onlys allows to do so at runtime without touching the experiment :p

**Comment by Lawouach** [comment]
Flag has been released in chaostoolkit-lib 1.11.0 and chaostoolkit 1.5.0

https://docs.chaostoolkit.org/reference/usage/run/#run-an-experiment-with-different-rollback-strategies
